reject-escalated-privileges-yaml/reject-escalated-privileges-yaml.rb

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env ruby
+
+require "json"
+require "octokit"
+require "yaml"
+
+require File.join(File.dirname(__FILE__), "github")
+
+# can expand this list spliting with spaces. e.g %w(cluster-admin root webops)
+STRING_LIST = %w[cluster-admin]
+
+def yaml_files(gh)
```

Comment:
I think the name "yaml_files" is a bit misleading here. It sounds like a function that should return all the yaml files in a PR.

The function actually returns "yaml files which elevate privileges" so I'd probably call it something like that - long names are good.